### ISSUE TITLE
StringSifter 

### ISSUE THREAD

**Issue by noraj** [BlackArch/blackarch#2829]
- **Name**: StringSifter 
- **Source URL**: https://github.com/fireeye/stringsifter

**Comment by dualfade** [comment]
Will PR

**Comment by dualfade** [comment]
In progress --
Errors with py deps during build --

(1/1) installing stringsifter                                                                                                                                               [##########################################################################################################] 100%
created virtual environment CPython3.8.6.final.0-64 in 254ms
  creator CPython3Posix(dest=/usr/share/stringsifter/venv, clear=False, global=False)
  seeder FromAppData(download=False, pip=latest, setuptools=latest, wheel=latest, via=copy, app_data_dir=/root/.local/share/virtualenv/seed-app-data/v1.0.1)
  activators BashActivator,CShellActivator,FishActivator,PowerShellActivator,PythonActivator,XonshActivator
Collecting lightgbm==2.1.2
  Downloading lightgbm-2.1.2-py2.py3-none-manylinux1_x86_64.whl (730 kB)
Collecting numpy==1.17.1
  Downloading numpy-1.17.1.zip (6.5 MB)
Collecting scikit-learn==0.21.3
  Downloading scikit-learn-0.21.3.tar.gz (12.2 MB)
    ERROR: Command errored out with exit status 1:
     command: /usr/share/stringsifter/venv/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-fal9r75b/scikit-learn/setup.py'"'"'; __file__='"'"'/tmp/pip-install-fal9r75b/scikit-learn/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' --no-user-cfg egg_info --egg-base /tmp/pip-pip-egg-info-j2pftitu
         cwd: /tmp/pip-install-fal9r75b/scikit-learn/

**Comment by dualfade** [comment]
On hold for now. Too many numpy dep errors. Multi version.
